### DISCUSSION TITLE
tailwind doesn't generate any json

### DISCUSSION THREAD

**Discussion by qramecourt** [tailwindlabs/tailwindcss#8276] [Help]
after following the steps for a CLI install, i had the bad surprise to see that neither packages.json nor packages-lock.json were generated. what can i do resolvethis issue?

**Comment by NovusOpus** [comment]
This files are not provided by Tailwind, those should be provided by the framework you are working with or you can generate one yourself with `npm init` command in your console this will ask you to fill stuff like project name, version license and more, you can setup those fields or  just press enter with every prompt to skip and use the default values.

So do `npm init` first to generate a packages.json file if you're not using any framework(nextjs, create-react-app, nuxt, laravell or others), then follow the steps for a CLI install.

**Reply by Webmo2004** [reply]
i still can not generate any of it the json package is not showing
